src/client/stdio.ts

```diff
@@ -113,16 +113,7 @@ export class StdioClientTransport implements Transport {
           stdio: ["pipe", "pipe", this._serverParams.stderr ?? "inherit"],
           shell: false,
           signal: this._abortController.signal,
-
-          // NB: The behavior of detached varies based on platform, and also
-          // is different based on whether the process is a Win32 Subsystem
-          // process or a Console Subsystem process. Strangely, the behavior
-          // of detached is almost 1:1 the opposite in Electron+Windows vs
-          // what is documented on the node.js website, and also is different
-          // based on whether you launch Electron in a development environment
-          // (i.e. via `electron-forge start`) vs a production environment
-          // (i.e. YourApp.exe).
-          detached: process.platform === "win32" && isElectron(),
+          windowsHide: process.platform === "win32" && isElectron(),
```

Comment:
TIOLI: Should this be an unconditional `true` now?

Reply:
Probably Yes but I wanted to be extra paranoid